analyzer/src/main/kotlin/Analyzer.kt

```diff
@@ -127,7 +130,7 @@ class Analyzer(private val config: AnalyzerConfiguration) {
         val vcs = VersionControlSystem.getCloneInfo(absoluteProjectPath)
         val repository = Repository(vcs, vcs.normalize(), repositoryConfiguration)
 
-        val run = AnalyzerRun(Environment(), config, analyzerResultBuilder.build())
+        val run = AnalyzerRun(startTime, Instant.now(), Environment(), config, analyzerResultBuilder.build())
```

Comment:
nit: This is a bit a magic number. Consider using a variable as you do for start time, o named params

Reply:
Done (in all three commits).

Reply:
Just for the record, I disagree here as it would mean you can never pass in the return value of a function as a parameter to another function (unless the first function is called like the parameter to the second function). Anyway, I would have preferred using named parameters here, otherwise it's a typical case where I'd inline an otherwise useless variable.

But I'm fine with merging this now, I just wanted to document my thoughts 😉

Reply:
> In my opinion in this review there were two options for getting rid of the magic number and both options are an improvement compared to how it was before. Can you agree with that?

Since you're asking, actually no, I would not agree with that. This case was simply too trivial IMO to improve readability significantly with either code change. But now that it's there, let's keep it. Personally I just wouldn't have spent any time on it.

Reply:
Oh, thanks. Now I'm sure I understood you correctly.